ui/apps/platform/src/Containers/BaseImages/BaseImagesPage.tsx

```diff
@@ -1,11 +1,135 @@
+                <Flex
+                    alignItems={{ default: 'alignItemsCenter' }}
+                    spaceItems={{ default: 'spaceItemsLg' }}
+                >
+                    <FlexItem flex={{ default: 'flex_1' }}>
+                        <Title headingLevel="h1">Base Images</Title>
+                        <Text>
+                            Manage approved base images for vulnerability tracking and
+                            layer-specific filtering
+                        </Text>
+                    </FlexItem>
+                    <FlexItem align={{ default: 'alignRight' }}>
+                        <Button variant="primary" onClick={handleOpenAddModal}>
+                            Add base image
+                        </Button>
+                    </FlexItem>
+                </Flex>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection className="pf-v5-u-display-flex pf-v5-u-flex-direction-column">
+                <Flex
+                    direction={{ default: 'column' }}
+                    spaceItems={{ default: 'spaceItemsLg' }}
+                    className="pf-v5-u-flex-grow-1"
+                >
+                    {deleteBaseImageMutation.isError && (
+                        <Alert
+                            variant="danger"
+                            isInline
+                            title="Error removing base image"
+                            component="p"
+                        >
+                            {getAxiosErrorMessage(deleteBaseImageMutation.error)}
+                        </Alert>
+                    )}
+
+                    {baseImagesRequest.isLoading && !baseImagesRequest.data && (
+                        <Bullseye>
+                            <Spinner aria-label="Loading base images" />
+                        </Bullseye>
+                    )}
+
+                    {baseImagesRequest.data !== undefined && (
+                        <BaseImagesTable
+                            baseImages={baseImages}
+                            onRemove={(baseImage) => handleRemoveBaseImage(baseImage.id)}
+                            isRemoveInProgress={deleteBaseImageMutation.isLoading}
+                        />
+                    )}
+                </Flex>
+            </PageSection>
+            <BaseImagesModal
+                isOpen={isAddModalOpen}
+                onClose={handleCloseAddModal}
+                onSave={() =>
+                    addBaseImageMutation.mutate({
+                        baseImageRepoPath: '',
+                        baseImageTagPattern: '',
+                    })
+                }
+                isSuccess={addBaseImageMutation.isSuccess}
+                isError={addBaseImageMutation.isError}
+                isSubmitting={addBaseImageMutation.isLoading}
+                error={isAxiosError(addBaseImageMutation.error) ? addBaseImageMutation.error : null}
```

Comment:
I think a non-axios error is unlikely, but do we need to explicitly exclude non-axios errors here?